core/optic/shared/src/main/scala/com/useoptic/diff/interpreters/Interpreter.scala

```diff
@@ -7,5 +7,6 @@ import scala.scalajs.js.annotation.{JSExportAll, JSExportDescendentClasses}
 
 
 trait InteractiveDiffInterpreter[T] {
-  def interpret(diff: T, interaction: HttpInteraction): Seq[InteractiveDiffInterpretation]
+  def interpret(diff: T, interactions: Vector[HttpInteraction]): Seq[InteractiveDiffInterpretation]
+  def interpret(diff: T, interaction: HttpInteraction): Seq[InteractiveDiffInterpretation] = interpret(diff, Vector(interaction))
```

Comment:
So it looks like you did follow the suggestion to keep the previous interface intact. However, you did it by mapping onto the one that takes many. There should be a distinction between interpreters that interpret based on one interaction only, and interpreters that are meant to interpret many interactions. So, all of the existing interpreters should revert to keeping the original interface that only takes 1 interpretation, instead of supporting the interface that takes multiple interpretations and then always choosing the first one.